src/main/java/com/FKFabian/medicalclinic/model/Visit.java

```diff
@@ -0,0 +1,59 @@
+package com.FKFabian.medicalclinic.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "VISITS")
+public class Visit {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false)
+    private LocalDateTime startingVisitTime;
+    @Column(nullable = false)
+    private LocalDateTime endingVisitTime;
+    @ManyToOne
+    @Column(name = "patient_id", nullable = false)
```

Comment:
tutaj pacjent musi przyjmować nulla przed jego dodaniem (jak to zostawimy to wywali nam sie aplikacja)